package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-native-fs2",
-  "version": "3.0.3",
+  "version": "3.0.4",
```

Comment:
May you remove this version change? We can tag a new version after some testing.

Reply:
Done